composer.json

```diff
@@ -15,6 +15,6 @@
         }
     ],
     "require": {
-        "drupal/enity": "8.1.x-dev"
+        "drupal/entity": "~8"
```

Comment:
Given that entity 8.x-1.x is different to 8.x-0.x we want to be more specific here.